daemon/daemon_unix.go

```diff
@@ -249,6 +249,11 @@ func parseSecurityOpt(securityOptions *container.SecurityOptions, config *contai
 				return fmt.Errorf("invalid --security-opt 2: %q", opt)
 			}
 			securityOptions.NoNewPrivileges = noNewPrivileges
+		case "cgroups":
```

Comment:
As this introduces a new option accepted by the API, can you add a bullet to the API version-history for v1.48? https://github.com/moby/moby/blob/20062ef3580a15e53c48e8e0e5aa2c43eb4982f1/docs/api/version-history.md#v148-api-changes

Something like (better suggestions welcome!)

```markdown
* `POST /containers/create` now accepts a `cgroups=rw` option in `HostConfig.SecurityOpt`
  to mount the container's cgroups writable. This provides a more granular
  approach than `HostConfig.Privileged`.
```

We should also include these options in our swagger / OpenAPI, but looks like we currently don't, so that's something we can do in a follow-up (I can create a tracking ticket for that).